app/client/public/icons/fullscreen.svg

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+<!-- Generator: Adobe Illustrator 18.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
+	"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg
+	version="1.1"
+	id="Capa_1"
+	xmlns="http://www.w3.org/2000/svg"
+	xmlns:xlink="http://www.w3.org/1999/xlink"
+	x="0px"
+	y="0px"
+	viewBox="0 0 53 53"
+	style="enable-background:new 0 0 53 53;"
+	xml:space="preserve"
+	fill="#fff"
+>
+<g>
+	<path
+			d="M52.923,0.618c-0.101-0.244-0.296-0.439-0.541-0.541C52.26,0.027,52.13,0,52,0H40c-0.552,0-1,0.448-1,1s0.448,1,1,1h9.586
+		L33.293,18.293c-0.391,0.391-0.391,1.023,0,1.414C33.488,19.902,33.744,20,34,20s0.512-0.098,0.707-0.293L51,3.414V13
+		c0,0.552,0.448,1,1,1s1-0.448,1-1V1C53,0.87,52.973,0.74,52.923,0.618z"
+		/>
+	<path
+			d="M18.293,33.293L2,49.586V40c0-0.552-0.448-1-1-1s-1,0.448-1,1v12c0,0.13,0.027,0.26,0.077,0.382
+		c0.101,0.244,0.296,0.439,0.541,0.541C0.74,52.973,0.87,53,1,53h12c0.552,0,1-0.448,1-1s-0.448-1-1-1H3.414l16.293-16.293
+		c0.391-0.391,0.391-1.023,0-1.414S18.684,32.902,18.293,33.293z"
+		/>
+	<path
+			d="M1,14c0.552,0,1-0.448,1-1V3.414l16.292,16.292c0.195,0.195,0.451,0.293,0.707,0.293s0.512-0.098,0.707-0.293
+		c0.391-0.391,0.391-1.023,0-1.414L3.414,2H13c0.552,0,1-0.448,1-1s-0.448-1-1-1H1C0.87,0,0.74,0.027,0.618,0.077
+		C0.373,0.179,0.179,0.373,0.077,0.618C0.027,0.74,0,0.87,0,1v12C0,13.552,0.448,14,1,14z"
+		/>
+	<path
+			d="M52,39c-0.552,0-1,0.448-1,1v9.586L34.707,33.292c-0.391-0.391-1.023-0.391-1.414,0s-0.391,1.023,0,1.414L49.586,51H40
+		c-0.552,0-1,0.448-1,1s0.448,1,1,1h12c0.13,0,0.26-0.027,0.382-0.077c0.244-0.101,0.439-0.296,0.541-0.541
+		C52.973,52.26,53,52.13,53,52V40C53,39.448,52.552,39,52,39z"
+		/>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>

```

Comment:
I have corrected this in the new PR from a different branch.

Reply:
For the Yaml Linter since we are using custom theme and not predefined ones, it is needed to modify the linter files which has some workers overriding the parser functions thus we need to have those files and also its required to be in the public folder in order to be accessible by the node package (https://www.npmjs.com/package/react-ace) that i am modifying. @rajdas98

Reply:
I have made necessary changes and raised a new PR for the same to the staging branch as we need to test the editor on litmuschaos/charthub.litmuschaos.io's staging server, before integrating it with litmus portal.